frontend/app/src/services/rotkehlchen-api.ts

```diff
@@ -141,43 +144,9 @@ export class RotkehlchenApi {
       .then(handleResponse);
   }
 
-  async allTrades(): Promise<ApiTrade[]> {
+  async allTrades(): Promise<Trade[]> {
```

Comment:
I did some changes and basically removed the "externalTrades"

Reply:
The idea is that instead of going by hand and doing all the conversion to bigNumber and camelcase we use the transformers instead. This way we don't need to have both typings or the converters. Of course we have to be careful with the usage of this and see API by API where it makes sense.